common.blocks/issue/issue.bemtree.js

```diff
@@ -12,35 +18,65 @@ block('issue').content()(function() {
             content: issue.created_from_now
         },
         {
-            elem: 'title',
-            content: {
-                block: 'link',
-                mix: { block: 'page', elem: 'link' },
-                url: '/' + issue.number + '/',
-                content: issue.title
-            }
-        },
-        issue.labels.length ? {
-            elem: 'labels',
-            content: issue.labels.map(function(label) {
-                return {
-                    elem: 'label',
-                    attrs: { style: 'border: 1px solid #' + label.color },
+            elem: 'container',
+            content: [
+                {
+                    elem: 'title',
                     content: {
                         block: 'link',
                         mix: { block: 'page', elem: 'link' },
-                        url: '/?labels=' + label.name,
-                        attrs: { style: 'color: #' + label.color },
-                        content: label.name
+                        url: '/' + issue.number + '/',
+                        content: issue.title
                     }
-                };
-            })
-        } : '',
-        {
-            elem: 'content',
-            content: issue.html
+                },
+                issue.labels.length ? {
+                    elem: 'labels',
+                    content: issue.labels.map(function(label) {
+                        return {
+                            elem: 'label',
+                            attrs: { style: 'border: 1px solid #' + label.color },
+                            content: {
+                                block: 'link',
+                                mix: { block: 'page', elem: 'link' },
+                                url: '/?labels=' + label.name,
+                                attrs: { style: 'color: #' + label.color },
+                                content: label.name
+                            }
+                        };
+                    })
+                } : '',
+                {
+                    elem: 'content',
+                    content: issue.html
+                },
+                {
+                    elem: 'spiner-overlay',
+                    content: {
+                        elem: 'loader',
+                        content: {
+                            block: 'spin',
+                            mix: { block: 'content', elem: 'spin' },
+                            mods: { theme: 'islands', size: 'xl', visible: false }
+                        }
+                    } 
+                }
+            ]
         },
-        this.data.url.pathname === '/' ? {
+        (issue['state'] === 'closed') ? {
+            block: 'button',
+            tag: 'div',
```

Comment:
Semantically it's not a button or link. So we avoid troubles with links preventdefault, or render problem with disabled button in old browser.